test/io/mandoline/impl/cache_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns io.mandoline.impl.cache-test
+  (:use clojure.test
+        clojure.test io.mandoline.backend.mem)
+  (:require
+    [clojure.test :refer :all]
+    [io.mandoline.backend.mem :as mem]
+    [io.mandoline.impl.protocol :as proto]
```

Comment:
If you're going to shorten requirements with namespace sharing, these two (`proto` and `cache`) seem like the place for it.